### PR TITLE
Make order of importers configurable

### LINKED ISSUE #1144: Make order of importers configurable
# Notes 

+ Some importers govern things like educator access to the tool, so they should probably run first — builds off of #1143 to make sure critical data is always imported

### PR DESCRIPTION
**Pull request by alexsoble**
# Issue

+ Fix #1144.

### CODE REVIEW

**Comment by alexsoble** [comment]
TODO: Handle case where we add an importer but don't add it to the `ORDER` list. Shouldn't blow up but should probably go last.

File: lib/tasks/import.thor [hunk 1]
```diff
@@ -52,6 +52,20 @@ class Import
...
+      EducatorSectionAssignmentsImporter => 2,
+      StudentsImporter => 3,
+      StudentSectionAssignmentsImporter => 4,
+      BehaviorImporter => 5,
+      AttendanceImporter => 5,
+      StudentSectionGradesImporter => 5,
+      X2AssessmentImporter => 6,
+      StarMathImporter::RecentImporter => 6,
+      StarReadingImporter::RecentImporter => 6,
+    }
```
Comment (kevinrobinson) [inline lib/tasks/import.thor]: Why not just a list?

Comment (jhilde) [reply lib/tasks/import.thor]: Why not a json configuration that could be transitioned to be per school district?

Comment (alexsoble) [reply lib/tasks/import.thor]: @kevinrobinson — I want to express the idea that some importers are app-critical and have dependencies — like educators which go before educator sections which go before students ... — while other importers are the same in terms of priority, like X2 assessments and STAR assessments. A list doesn't express that. 

@jhilde — My vote would be let's see what Fall River's data looks like before making this configurable. :D

Comment (kevinrobinson) [reply lib/tasks/import.thor]: @alexsoble Sounds great!  Using `priority` instead of `order` might capture that.

**Comment by alexsoble** [comment]
@kevinrobinson 👍🏽👍🏽

**Comment by alexsoble** [comment]
Adding some sorting-specific specs, those are 💚 . Also exercised the import process locally, it ran good and in the correct order. 🎉

**Pull request merged**